hack/make/ubuntu

```diff
@@ -150,6 +158,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
Actually, checking back with the debian policy manual, I really prefer this is a Recommends. It's not simply suggested to use an LSM with Docker. Recommends is also what we use for the debian packages.